### ISSUE TITLE
a11y: Alternative texts for images

### ISSUE THREAD

**Issue by rewodev** [quilljs/quill#4206]
In terms of accessibility there should be the possibility to set alternative texts for images. In the current version (2.0.2), this is not possible.

**Steps for Reproduction**

1. Visit [Quill Playground](https://quilljs.com/playground/snow)
2. Insert an image into the editor
3. Try to set an alternative text for the inserted image

**Expected behavior**:

An alternative text can be set.

**Actual behavior**:

An alternative text cannot be set.
